contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -125,14 +124,18 @@ abstract contract CompoundingValidatorManager is Governable, Pausable {
         EXITED, // The validator has been verified to have a zero balance
         REMOVED // validator has funds withdrawn to the EigenPod and is removed from the SSV
     }
+
+    // For future use
+    uint256[49] private __gap;
```

Comment:
this is a new contract so I've started the gap at 50

Reply:
In some contracts we did it in a way where even at contract creation the slots + the gap would make up the number 50. This way each element in inheritance would always take up multiples of 50  or a 100. I personally prefer it that way. What do you think?